common/components/register_model/spec.yaml

```diff
@@ -0,0 +1,63 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+name: register_model
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: Register model
+description:
+  Register a model to a workspace or a registry. The component works on compute with [MSI](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-manage-compute-instance?tabs=python) attached.
+
```

Comment:
How do we make this work in case of import flow submitted through UI?

Reply:
Let us check the "register" flow on the UI & how auth works in that case. 

This could potentially be a blocker for using registration in the import flow.

Cc: @ManojBableshwar @swatig007

Reply:
Created an internal WI to track this. Thank you